### DISCUSSION TITLE
Anyone know any good brush packs for Toon Squid?

### DISCUSSION THREAD

**Discussion by Mirakelpung** [keiwando/toonsquid#461] [General]
Do you guys know any good places for getting brushes compatible with Toonsquid? Or any particular brush packs that just works well with this program.

I recently purchased Aaron Blaise's pencil pack and I've enjoyed using those in Procreate, but sadly the main brush pack of the two sets you get don't seem compatible with Toonsquid and the pencils I usually use in Procreate is one of the default of that program not available in Toonsquid. So I'm struggling aa bit finding brushes I find comfortable to work with in Toonsquid.

**Comment by Kosdaos1** [comment]
Try kyle's brushes. These are sets of excellent brushes for photoshop, but they work well in Toonsquid too

**Comment by Mirakelpung** [comment]
Thanks dude, I will look that up

**Comment by PencilMarkz** [comment]
I know this is an old post but I wanted to recommend:
https://bbrushes.gumroad.com/l/toonsquid

It's a set designed especially for ToonSquid so it's way more functional than any PS brushes you import. It's really helped me up my game, I highly recommend
